docs/transpiling.md

```diff
@@ -1,52 +1,53 @@
 
 For example, you might want to use [Babel](https://babeljs.io/):
 
 - In a configuration file `{ requireModule: ['@babel/register'] }`
 - On the CLI `$ cucumber-js --require-module @babel/register`
 
-This would mean any support code loaded with the `require` option would be transpiled first.
+This would mean any support code loaded with the `require` option would be transpiled first then loaded into Cucumber.
 
 ## TypeScript
 
-Your `tsconfig.json` should have these `compilerOptions` on:
+Your `tsconfig.json` should have these `compilerOptions`:
 
 ```json
 "allowSyntheticDefaultImports": true,
 "resolveJsonModule": true,
+
 ```
 
-Other than that, a pretty standard TypeScript setup should work as expected.
+Typescript's output must match Cucumber's expected import, and this is controlled by the "module" option in the `tsconfig.json`.  The default is "CommonJS", but if your project is setup to be an ESM project then Typescript will need to output some generation of ES code to Cucumber. If you aren't sure use "ESNext".
 
-You'll also need to specify where your support code is, since `.ts` files won't be picked up by default.
+Cucumber doesn't load `*.ts` files by default. The glob pattern `step-definitions/**/*.ts` will load them from the default step definition location.
+
+Other than that, a pretty standard TypeScript setup should work as expected.
 
 ### With ts-node
 
-If you are using [ts-node](https://github.com/TypeStrong/ts-node):
+[TS-Node](https://github.com/TypeStrong/ts-node): is the one of the most popular ways to load TypeScript files. 
 
 - In a configuration file `{ requireModule: ['ts-node/register'], require: ['step-definitions/**/*.ts'] }`
 - On the CLI `$ cucumber-js --require-module ts-node/register --require 'step-definitions/**/*.ts'`
 
-#### ESM
+If you are using ts-node in a CommonJS project then this configuration will work, but if you have an ESM project there are additional steps.
 
-For ESM projects, you can use `ts-node`'s ESM loader and then `import` your TypeScript files:
+* Set TypeScript to export to an ES format such as "ESNext" using the `ts-config.json` file. 
+* Then use `ts-node`'s ESM loader to import your TypeScript. 
 
-```shell
-$ NODE_OPTIONS="--loader ts-node/esm" cucumber-js --import 'step-definitions/**/*.ts'
-```
-
-Don't forget to set your `tsconfig.json` to emit JavaScript with `import` and `export` statements:
+That last step requires setting an environment variable. The cleanest way to do this is to include the [cross-env](https://www.npmjs.com/package/cross-env) package with `npm i -D cross-env`. With that package installed make the following change to your npm test script invocation in the package.json file:
```

Comment:
I don't think we should be prescriptive of using this particular library/approach. By all means mention it as an option but examples should be as vanilla as possible really.

Reply:
Done.